src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/NomisSyncResourceTest.kt

```diff
@@ -409,237 +409,6 @@ class NomisSyncResourceTest : SqsIntegrationTestBase() {
           .bodyValue(jsonString(updatedSyncRequest))
           .exchange()
           .expectStatus().isCreated
-          .expectBody().json(
```

Comment:
maybe these body assertions can become something like
```kotlin
.expectBody().consumeWith {
  val reportId = objectMapper.readValue(it.responseBody, UUID::class.java)
  assertThat(reportId).isNotNull()
}
```
so that we actually ensure a uuid is returned?

Reply:
also, the problem with removing the JSON response assertions is that now we have nothing assuring the right stuff got saved in the db

Reply:
Yes I realise that - feel like that test should be elsewhere